content/non-web-ict.md

```diff
@@ -57,7 +59,7 @@ Update in progress: The WCAG2ICT Task Force is currently updating WCAG2ICT to al
 
 *This document:* **WCAG2ICT** is a W3C Working Group Note that is informative — it is not normative and does not set requirements. It provides guidance on the application of WCAG 2.0 principles, guidelines, and success criteria to non-web documents and software. WCAG2ICT was developed to provide an authoritative interpretation of how WCAG 2.0 can apply in different contexts. For information on the scope of WCAG2ICT, see the [WCAG2ICT Introduction](https://www.w3.org/TR/wcag2ict/#intro).
 
-*Note:* **WCAG2ICT is still relevant for later versions of WCAG 2**. For example, WCAG 2.1 does not change the requirements of WCAG 2.0 that are addressed in WCAG2ICT; WCAG 2.1 adds additional requirements. The WCAG2ICT Task Force is currently updating WCAG2ICT to cover WCAG 2.1 and WCAG 2.2.
+*Update in progress:* @@ link to2.2  WD @@ draft includes guidance for WCAG 2.1 success criteria and glossary terms. We will add guidance for WCAG 2.2 success criteria and address open issues in later drafts.
```

Comment:
```suggestion
*Update in progress:* [WCAG2ICT](https://wwww.w3.org/TR/2023/DNOTE-wcag2ict-20230815/) draft includes guidance for WCAG 2.1 success criteria and glossary terms. We will add guidance for WCAG 2.2 success criteria and address open issues in later drafts.
```

Assuming this is where we want to point to here as well.

Reply:
short URI instead of dated URI so it doesn't get stale